webpack.config.js

```diff
@@ -343,8 +372,12 @@ function getWebpackConfig( {
 				minify: ! isDevelopment,
 			} ),
 			new AssetsWriter( {
```

Comment:
Could we add `isCalypsoClient &&`  check here not to write these when building using the SDK builder?

You can test with:
```
npm run sdk -- gutenberg client/gutenberg/extensions/presets/jetpack/ --output-dir=../somewhere-in-your-system --watch
```

https://wpcalypso.wordpress.com/devdocs/docs/sdk.md

Reply:
Yup, not a problem, done! Is there something in this PR that causes this to become an issue?

Reply:
Not at all and I recognise you didn't change this behaviour from how it was before — it's just unnecessary step in the SDK build and this seems like a good moment to add this check.

Potential annoyances could only be caused during development by running SDK build in watch mode while running Calypso, in which case assets from the SDK build could end up used in Calypso.

Reply:
Gotcha, happy to remove unnecessary work from the build! 👍